server/controllers/users.js

```diff
@@ -34,19 +59,25 @@ export const exists = async (req, res) => {
 export const signin = (req, res, next) => {
```

Comment:
At this stage, can you have a specific `signin` endpoint for blockstack? And use a dedicated `exists` method?

To minimize footprint in `server/controllers/users.js`, supporting functions may be loaded from `lib/blockstack.js`